pkg/virt-controller/services/template.go

```diff
@@ -1155,11 +1156,11 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 		compute.Env = append(compute.Env,
 			k8sv1.EnvVar{
 				Name:  "XDG_CACHE_HOME",
-				Value: varRun,
+				Value: homeQemu,
 			},
 			k8sv1.EnvVar{
 				Name:  "XDG_CONFIG_HOME",
-				Value: varRun,
+				Value: homeQemu,
 			},
 			k8sv1.EnvVar{
 				Name:  "XDG_RUNTIME_DIR",
```

Comment:
Is there a specific reason to leave `XDG_RUNTIME_DIR` set to `/var/run` while changing all other `XDG_*` vars? This just looks a bit strange to me since the current permissions will not allow any user except root to write there:
```
bash-4.4$ ls -la /var/run/
total 4
drwxr-xr-x. 1 root root  108 Apr 29 08:45 .
dr-xr-xr-x. 1 root root   74 Apr 29 08:45 ..
...
bash-4.4$ touch /var/run/test 
touch: cannot touch '/var/run/test': Permission denied
```
For me it seems that `XDG_RUNTIME_DIR` is not really used. Maybe makes sense to drop it then?

Reply:
Yes to that last part! If we move everything to /home/qemu, migration fails for obscure reasons, even with a symlink from /var/run/libvirt to /home/qemu/libvirt.
Since nothing ever seems to get written at the root of XDG_RUNTIME_DIR and since /var/run/libvirt is chmoded 777 we're all good.
However, swtpm does write to the root of XDG_CONFIG_HOME so it definitely had to be moved to avoid giving qemu full read-write access to /var/run.

Reply:
I see. Makes sense. Interesting why migration may fail with the symlink... Anyway, thanks for the details.